src/main/groovy/com/avioconsulting/mule/designcenter/HttpClientWrapper.groovy

```diff
@@ -1,5 +1,8 @@
 package com.avioconsulting.mule.designcenter
 
+import com.avioconsulting.mule.designcenter.api.models.credentials.ConnectedAppCredential
+import com.avioconsulting.mule.designcenter.api.models.credentials.Credential
+import com.avioconsulting.mule.designcenter.api.models.credentials.UsernamePasswordCredential
```

Comment:
Why `designcenter` package? This model applies to the authentication of anypoint platform. Maybe we should use `accessmanagement` or just `anypoint`?

Reply:
It's a very valid observation @manikmagar, I do think it should go to `anypoint` too now that you mention it.